datasets/simulated.py

```diff
@@ -28,8 +25,9 @@ def __init__(self, n_samples=10, n_features=50, rho=0, random_state=27):
     def get_data(self):
         rng = np.random.RandomState(self.random_state)
 
-        X, y, _ = make_correlated_data(self.n_samples, self.n_features,
-                                       rho=self.rho, random_state=rng)
+        X, y, _ = make_correlated_data(
```

Comment:
we try to avoid formatting change like these which make the diff longer than it could be. some people don't use black :)